operators/merge_from_scene.py

```diff
@@ -0,0 +1,53 @@
+import bpy
+from .utils.doc import doc_name, doc_idname, doc_brief, doc_description
+
+
+class MergeFromSceneStrip(bpy.types.Operator):
+    """
+    Merge into the current scene meta data from a scene strip re-creating strips and markers.
+    """
+    doc = {
+        'name': doc_name(__qualname__),
+        'demo': '',
+        'description': doc_description(__doc__),
+        'shortcuts': [],
+        'keymap': 'Sequencer'
+    }
+    bl_idname = doc_idname(doc['name'])
+    bl_label = doc['name']
+    bl_description = doc_brief(doc['description'])
+    bl_options = {"REGISTER", "UNDO"}
+
+    @classmethod
+    def poll(cls, context):
+        return context.scene.sequence_editor.active_strip.type == 'SCENE'
+
+    def execute(self, context):
+        strip = context.scene.sequence_editor.active_strip
+        strip_scene = strip.scene
+        start_scene = context.screen.scene
+
+        self.merge_strips(strip_scene, start_scene)
+        self.merge_markers(strip_scene, start_scene)
+
```

Comment:
I think it should optionally delete the target scene. That's what merging means - you're looking to merge 2 scenes into one

Reply:
You can use a BoolProperty for that